inference-engine/src/gna_plugin/frontend/scale_factor_calc.hpp

```diff
@@ -199,9 +199,17 @@ class ScaleFactorPerLayer<InferenceEngine::CNNLayer *> {
 
         if (cnnLayer->type == "Const") {
             auto blob = cnnLayer->blobs["custom"];
-            if (blob->getTensorDesc().getPrecision() == InferenceEngine::Precision::FP16) {
+            auto blob_precision = blob->getTensorDesc().getPrecision();
+
+            if (blob_precision != InferenceEngine::Precision::FP32 && blob_precision != InferenceEngine::Precision::FP16) {
```

Comment:
looks correct for Int32 const blobs we might add diagonal layer in pasmanager, so those blobs aso will be consumed as is with scale factor of 1